test/suite.bats

```diff
@@ -62,3 +62,15 @@ fixtures suite
   [ "${lines[5]}" = "begin 3 quasi-truth" ]
   [ "${lines[6]}" = "not ok 3 quasi-truth" ]
 }
+
+@test "recursive support (short option)" {
+  run bats -r "${FIXTURE_ROOT}/recursive"
+  [ $status -eq 0 ]
+  [ "${lines[0]}" = "1..2" ]
```

Comment:
Can you also check that the following `lines` emit the tests in the expected order per `find | sort`? I'm pretty sure we'll see:

```
ok 1 another passing test
ok 2 a passing test
```